runtime/participant-runtime.ss

```diff
@@ -275,7 +276,7 @@
        (error "agreements don't match" (.@ self agreement) agreement))
      (set! (.@ self timer-start) (.@ agreement options maxInitialBlock))
      (set! (.@ self first-unprocessed-block) (.@ contract-config creation-block))
-     (interpret-current-code-block self))
+     (void))
```

Comment:
The call to `interpret-current-code-block` is now in 2 of the 4 branches, instead of in all 4.

The 4 branches I'm referring to are:
 - (a) Active-participant with no contract yet
 - (b) Active-participant with an existing contract
 - (c) Passive-participant with a handshake
 - (d) Passive-participant with an existing contract

The calls in branches (b) and (d) are the same as always, the calls in branches (a) and (c) have been removed, and should only be put back when we determine what conditions it is safe to reinstate the first-transaction optimization, and guard it with that.

However, I've been working on de-prioritizing the distinction between active and passive participants here, to simplify those 4 branches into just two, based on only whether they have an existing contract or not.